gcp/v2/asm/istio-operator.yaml

```diff
@@ -25,14 +25,14 @@ spec:
       istio-ingressgateway:
         type: NodePort
     global:
-      meshID: "project-id_us-central1_name" # {"$kpt-set":"asm-mesh-id"}
+      meshID: "proj-147474701642" # {"$kpt-set":"asm-mesh-id"}
```

Comment:
You probably forgot to reset the id?

Reply:
Good catch. I updated the gcp-reset make command and reran it. A bunch of other values got reset as well.